examples/pytorch/gat/train.py

```diff
@@ -53,37 +53,38 @@ def __init__(self,
         self.residual = residual
         if residual:
             if in_dim != out_dim:
-                self.residual_fc = nn.Linear(in_dim, num_heads * out_dim, bias=False)
-                nn.init.xavier_normal_(self.fc.weight.data, gain=1.414)
+                self.res_fc = nn.Linear(in_dim, num_heads * out_dim, bias=False)
```

Comment:
While this may add more model capacity, I thought they did `nn.Linear(in_dim, out_dim, bias=False)` as in [L30](https://github.com/PetarV-/GAT/blob/master/utils/layers.py#L30)?